app/build.gradle

```diff
@@ -48,6 +48,11 @@ android {
     kotlinOptions {
         jvmTarget = "1.8"
     }
+    lintOptions {
+        disable 'GoogleAppIndexingWarning'
```

Comment:
I don't prefer to ignore `GoogleAppIndexingWarning`, the purpose of `sunflower` project is a demo for different Android features of jetpack or androidX, the  `GoogleAppIndexingWarning` can be one part of them. 

Let the warning there is OK.

Reply:
My intension was to force warnings as errors by `warningsAsErrors` attribute, with that all changes has verify by CI on PR forcing to failure if was introduces new lint warnings (forcing contributors to check them). That was the only reason for why I added the disable 'GoogleAppIndexWarning'.

If you consider that isn't necessary I remove that also.  Please confirm me, for make these changes on PR

Reply:
cc @tiembo